### PR TITLE
Improve JSF's testAjaxInputFile 

### PR DESCRIPTION
**Pull request by volosied**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

Follow up for #28698

### CODE REVIEW

**Comment by volosied** [comment]
!build

File: dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22InputFile.war/src/com/ibm/ws/jsf22/fat/input/jsf22/FileBean.java [hunk 1]
```diff
@@ -51,6 +51,13 @@ public Part getFile() {
         return file;
     }
 
+    public long getFileSize(){
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22InputFile.war/src/com/ibm/ws/jsf22/fat/input/jsf22/FileBean.java:54]: Copyright update?

File: dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22InputFile.war/src/com/ibm/ws/jsf22/fat/input/faces40/FileBean.java [hunk 2]
```diff
@@ -58,4 +58,12 @@ public void setFile(Part file) {
     public String getFileContents() {
         return fileContents;
     }
+
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22InputFile.war/src/com/ibm/ws/jsf22/fat/input/faces40/FileBean.java:61]: Copyright update?

File: dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22InputFile.war/resources/AjaxFileUploadTest.xhtml [hunk 3]
```diff
@@ -23,9 +23,11 @@
         <h:form id="form1" enctype="multipart/form-data">
             <p>Enter a file to upload:</p>
             <h:inputFile id="file1" value="#{fileBean.file}">
-                <f:ajax event="change"/>
+                <f:ajax render="size" event="change"/>
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22InputFile.war/resources/AjaxFileUploadTest.xhtml:26]: No copyright update to the date range

File: dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22InputFileTests.java [hunk 4]
```diff
@@ -194,6 +194,10 @@ public void testAjaxInputFile() throws Exception {
         //Ensure the correct content type is used. 
         assertNotNull("The 'multipart/form-data; boundary=' content type was not found!", jsfTestServer2.waitForStringInTraceUsingMark(".*multipart/form-data; boundary=.*"));
 
+        Log.info(c, name.getMethodName(), page.getPageSource());
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22InputFileTests.java:197]: Copyright date range update?

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
#libby ([view Open Liberty Delivery Requirements Verification](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=b1df6673-c0b9-4a55-aee4-7356886bb38b) - :white_check_mark: completed successfully!)

_Note: Target locations of links might be accessible only to IBM employees._

**Pull request merged**